packages/mui-material/src/Popper/Popper.tsx

```diff
@@ -49,11 +49,13 @@ const Popper = React.forwardRef(function Popper(
   ref: React.ForwardedRef<HTMLDivElement>,
 ) {
   const theme = useTheme<{ direction?: Direction }>();
-  const { components, componentsProps, slots, slotProps, ...other } = useThemeProps({
+  const props = useThemeProps({
```

Comment:
The API docs generation script requires the `props` variable to be destructured to work properly.

Reply:
Yep, I remember this restriction from when I was adding the `useThemeProps`. This is why the param in `React.forwardRef` are called `inProps` :))